docs/user/custom-resources/apigateway/apigateway-v1beta2-adr.md

```diff
@@ -0,0 +1,148 @@
+| **rules.mutators**              |  **NO**   | Specifies the list of the request mutators.                                                                                                                                                                                                                                                                                                  | Currently, the `Headers` and `Cookie` mutators are supported. For more information, see the [documentation](https://github.com/kyma-project/api-gateway/blob/main/docs/user/custom-resources/apirule/04-40-apirule-mutators.md). |
+| **rules.accessStrategies**      |  **YES**  | Specifies the list of access strategies.                                                                                                                                                                                                                                                                                                     | Supported are `extAuth` and `jwt` as [Istio](https://istio.io/latest/docs/tasks/security/authorization/authz-jwt/) access strategy.                                                                                              |
+| **rules.timeout**               |  **NO**   | Specifies the timeout, in seconds, for HTTP requests made to **spec.rules.path**. Timeout definitions set at this level take precedence over any timeout defined at the **spec.timeout** level.                                                                                                                                              | The maximum timeout is limited to 3900 seconds (65 minutes).                                                                                                                                                                     |
+
+### Examples
+
+- Multiple hosts with external authorizers:
+```yaml
+apiVersion: gateway.kyma-project.io/v1beta2
+kind: APIRule
+metadata:
+  name: service-config
+spec:
+  gateway: kyma-system/kyma-gateway
+  hosts:
+    - api1.example.com
+    - api2.example.com
+  service:
+    name: httpbin
+    port: 8000
+  rules:
+    - path: /headers
+      methods: ["GET"]
+      accessStrategy:
+        extAuth:
+          - name: oauth2-proxy
+            restrictions:
+              authentications:
+                - issuer: https://example.com
+                  jwksUri: https://example.com/.well-known/jwks.json            
+              authorizations:
+                - audiences: ["app1"]
+          - name: geo-blocker
+```
+
+- One host with jwt:
+```yaml
+apiVersion: gateway.kyma-project.io/v1beta2
+kind: APIRule
+metadata:
+  name: service-config
+spec:
+  gateway: kyma-system/kyma-gateway
+  hosts:
+    - app1.example.com
+  service:
+    name: httpbin
+    port: 8000
+  rules:
+    - path: /headers
+      methods: ["GET"]
+      accessStrategy:
```

Comment:
mixture of `jwt` and `extAuth` allowed? can you have more than one `jwt`s and/or `extAuth`s ? validation?

Reply:
It's not allowed, that's why it's not a list and the name is now singular.